documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -58,266 +64,245 @@ Run the example codes below in a `mongo` session to:
+    === "SSL/TLS secured MongoDB"
+        ```sh
+        pmm-admin add mongodb \
+        --username=pmm \
+        --password=your_secure_password \
+        --tls \
+        --tls-certificate-key-file=/path/to/client.pem \
+        --tls-certificate-key-file-password=cert_password \  # If certificate has password
+        --tls-ca-file=/path/to/ca.pem \
+        --authentication-mechanism=MONGODB-X509 \  # For X.509 authentication
+        --authentication-database=$external      # For X.509 authentication
+        ```
 
-3. Click **Add service**.
+    When successful, PMM Client will print `MongoDB Service added` with the service's ID and name. Use the `--environment` and `--custom-labels` options to set tags for the service to help identify them.
 
-![!](../../../images/PMM_Add_Instance_MongoDB_TLS.jpg)
+    !!! hint alert alert-success "Tips"
+        - When adding nodes to a sharded cluster, ensure to add each node separately using the `--cluster mycluster` option. This allows the [MongoDB Cluster Summary](../../../reference/dashboards/dashboard-mongodb-cluster-summary.md) dashboard to populate correctly. 
+        - You can also use the `--replication-set` option to specify a replication set, although they are automatically detected. For instance, you can use `--replication-set config` for your config servers; `--replication-set rs1` for your servers in the first replica set, `--replication-set rs2` for your servers in the second replica set, and so on.
+        - When running mongos routers in containers, specify the `diagnosticDataCollectionDirectoryPath` to ensure that pmm-agent can properly capture mongos metrics. For example: `mongos --setParameter diagnosticDataCollectionDirectoryPath=/var/log/mongo/mongos.diagnostic.data/`
 
-### On the command line
+## Verify MongoDB Service Configuration
 
-Look for your service in the output of this command.
+After adding MongoDB service to PMM, verify that it's properly configured and collecting data. This ensures your monitoring setup is working correctly.
+{.power-number}
 
-```sh
-pmm-admin inventory list services --service-type=mongodb
-```
+1. Check service registration:
 
-### Check data
+    === "Via UI"
+        To check the service from the UI:
 
-To check the data:
-{.power-number}
+        - Select **PMM Configuration > Inventory > Services**. 
+        - Find your MongoDB service in the list and verify it shows "Active" status.
+        - Verify the **Service name**, **Addresses**, and other connection details are correct.
+        - In the **Options** column, expand the **Details** section to check that agents are properly connected.
 
-1. Open the **MongoDB Instances Overview** dashboard.
-2. Set the **Service Name** to the newly-added service.
+    === "Via CLI"
+        Look for your service in the output of this command:
 
-#### Query Analytics
+        ```sh
+        pmm-admin inventory list services --service-type=mongodb
+        ```
```

Comment:
Tbh, I'm not sure if its' correct way to check, I'd check with pmm-admin list. I couldn't find any other places we mention "pmm-admin inventory" command and how to use it. Using pmm-admin list command can help user to see any problem with added MongoDB service, like not started agents, profiling is not configured